### DISCUSSION TITLE
clean-slate stable version up

### DISCUSSION THREAD

**Discussion by AveYo** [AveYo/MediaCreationTool.bat#133] [Announcements]
it's finally in the form I've envisioned it when I started the last rework  
hopefully I haven't misses any outstanding issues and I can go back to once in a blue moon maintenance ;)

**Comment by AveYo** [comment]
>_If you had no success launching the script so far, this latest version will work_  

so much confidence ;)

**Reply by AveYo** [reply]
anyone?

**Reply by yodaminium** [reply]
Virus detects malicious script. Blocks it from running.
It auto-downloads the latest version and prepares for upgrade - why?
(this is Microsoft behaviour, I am trying to avoid that-MOST people using these scripts want to CHOOSE their version not have it auto-upgrade to the latest ISO-I could do that with MS's media creation tool, negating the necessity of your script)

Does it still work since MS blocked ISO downloads in this manner from the public-or whatever they did to cause other tools of the same = ilk to stop working? IE - Rufus, Techbench UUDump etc,etc.

I have rem'd out the options for auto-upgrade and it still tries to do it, just before it fails to run due to the virus detection from my antivirus.
